hex/web/blueprints/post_blueprint.py

```diff
@@ -28,4 +29,11 @@ def post_detail(post_id: int) -> Response:
         post = get_post.execute(post_id=post_id)
         return jsonify(post.to_dict())
 
+    @post_blueprint.errorhandler(PostNotFound)
```

Comment:
I would probably implement this inline as a `try/except` around the `get_post.execute` call so it's explicit where the error can come from